packages/image_picker/image_picker/ios/Classes/FLTImagePickerPlugin.m

```diff
@@ -37,16 +37,18 @@ @interface FLTImagePickerPlugin () <UINavigationControllerDelegate,
 
 @property(strong, nonatomic) PHPickerViewController *pickerViewController API_AVAILABLE(ios(14));
 
+@property(strong, nonatomic) UIImagePickerController *imagePickerController;
+
+@property(strong, nonatomic) NSMutableArray<UIImagePickerController *> *imagePickerControllerOverrides;
```

Comment:
Needs a declaration comment, per style.

Reply:
Added the declaration comments as required by style.